dev/io.openliberty.cdi.3.0.internal_fat/fat/src/io/openliberty/cdi30/internal/fat/tests/CDI30BeansXMLTest.java

```diff
@@ -39,10 +41,11 @@ public class CDI30BeansXMLTest extends FATServletClient {
 
     private static final String BEANS_XML_APP_NAME = "CDI30BeansXMLApp";
 
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat(SERVER_NAME, EERepeatActions.EE9, EERepeatActions.EE10);
```

Comment:
This would run EE9 in LITE and EE10 in FULL ... which is the opposite to what we do elsewhere? Should it be switched?

Reply:
we'll review LITE vs FULL later